3rdparty/llvm.cmake

```diff
@@ -53,7 +56,7 @@ if(WITH_LLVM)
 		endif()
 	endif()
 
-	set(LLVM_LIBS LLVMMCJIT LLVMX86CodeGen LLVMX86AsmParser)
+	set(LLVM_LIBS LLVMMCJIT LLVMX86CodeGen LLVMX86AsmParser LLVMIntelJITEvents)
```

Comment:
Can you make this conditional like above? On FreeBSD with LLVM enabled (not on CI yet) I get
```
ld: error: unable to find library -lLLVMIntelJITEvents
```